rke/k8s_rke_system_images.go

```diff
@@ -7670,6 +7670,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out-of-band post v2.6.7
+		"v1.22.13-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.3",
+			Kubernetes:                "rancher/hyperkube:v1.22.13-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.87",
+			NginxProxy:                "rancher/rke-tools:v0.1.87",
+			CertDownloader:            "rancher/rke-tools:v0.1.87",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.87",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.17.4",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.17.4",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.17.4",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.3",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.21.1",
+			CalicoCNI:                 "rancher/calico-cni:v3.21.3-rancher1",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.21.1",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.21.1",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.21.1",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.21.1",
+			CanalCNI:                  "rancher/calico-cni:v3.21.1-rancher1",
```

Comment:
This needs to be updated to `rancher/calico-cni:v3.21.3-rancher1` as well.

Reply:
Updated